package.json

```diff
@@ -54,15 +54,15 @@
   },
   "devDependencies": {
     "@react-three/drei": "^9.68.2",
-    "@react-three/fiber": "^8.13.0",
+    "@react-three/fiber": "9.0.0-rc.1",
     "@storybook/addon-essentials": "^7.0.10",
     "@storybook/addon-interactions": "^7.0.10",
     "@storybook/addon-links": "^7.0.10",
     "@storybook/blocks": "^7.0.10",
     "@storybook/react": "^7.0.10",
     "@storybook/react-vite": "^7.0.11",
     "@storybook/testing-library": "^0.0.14-next.2",
-    "@types/react": "^18.2.0",
+    "@types/react": "^18.3.12",
     "@types/react-dom": "^18.2.1",
```

Comment:
BTW this should be:

```json
"@types/react": "npm:types-react@19.0.0-rc.1",
"@types/react-dom": "npm:types-react-dom@19.0.0-rc.1",
```

which may also need:

```json
"overrides": {
  "@types/react": "npm:types-react@19.0.0-rc.1",
  "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1"
}
```

This is detailed in https://react.dev/blog/2024/04/25/react-19-upgrade-guide, and we have our own https://github.com/pmndrs/react-three-fiber/blob/v9/docs/tutorials/v9-migration-guide.mdx.

Reply:
Done!